airflow/config_templates/config.yml

```diff
@@ -2052,6 +2052,22 @@ webserver:
       type: boolean
       example: ~
       default: "False"
+    max_form_memory_size:
+      description: |
+        The maximum size in bytes any non-file form field may be in a multipart/form-data body.
+        If this limit is exceeded, a 413 RequestEntityTooLarge error is raised by webserver.
+      version_added: 2.10.5
```

Comment:
@utkarsharma2 @ephraimbuddy are we going to have a 2.10.5?

Reply:
Yes